src/lib/components/TokenSelect/index.tsx

```diff
@@ -140,6 +139,7 @@ export default memo(function TokenSelect({ value, collapsed, disabled, onSelect
       <TokenButton value={value} collapsed={collapsed} disabled={disabled} onClick={onOpen} />
       {open && (
         <Dialog color="module" onClose={() => setOpen(false)}>
+          <DialogHeader title={<Trans>Select a token</Trans>} />
```

Comment:
Moving the header here breaks the TokenSelect.fixture.tsx. Instead, how about a new

```
function TokenSelectDialog() {
  return (
    <>
      <Header ...>
      <TokenSelectBody />
    <>
  )
}
```

with the existing logic moved into `TokenSelectBody` (or something similar).